rest-fights/pom.xml

```diff
@@ -132,6 +132,46 @@
       <groupId>io.quarkus</groupId>
       <artifactId>quarkus-liquibase-mongodb</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-stork</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.stork</groupId>
+      <artifactId>stork-service-discovery-static-list</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.stork</groupId>
+      <artifactId>stork-service-discovery-kubernetes</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.stork</groupId>
+      <artifactId>stork-service-discovery-eureka</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.smallrye.stork</groupId>
```

Comment:
since consul and eureka are not used, only mentioned in the docs, what's the reason for including them here?

Reply:
Since this is a demo/sample app I thought including them so that anyone "playing" with the app could simply be able to configure it to use eureka or consul without having to modify code/build, container image, etc, might be a good thing. This way users can just start experimenting.

Reply:
Is having all of these extensions present going to cause a problem? Like I mentioned, having them already included allows users to "play" with the app in various configurations without having to build new container images.

Reply:
I don't think it's a problem for an example application

Reply:
I don't have permission to mark the discussion as resolved but it's resolved now :)

Reply:
Thank you very much for the review!